server/controllers/CommentLikesController.js

```diff
@@ -65,6 +63,58 @@ class CommentLikesController {
     }).catch(next);
   }
 
+  /**
+   * @description Delete a like for a comment
+   *
+   * @param {Object} req - HTTP Request
+   * @param {Object} res - HTTP Response
+   * @param {*} next - Next function
+   *
+   * @return {Object} Returned object
+   */
+  static unLikeComment(req, res, next) {
+    const { id } = req.params;
+    const { userId } = req.decoded;
+    Comment.findOne({
+      where: {
+        id: {
+          [Op.eq]: id,
+        }
+      }
+    }).then((comment) => {
+      if (!comment) {
+        return res.status(404).json({
+          message: 'Comment does not exist'
+        });
+      }
+
+
+      CommentLike.findOne({
+        where: {
+          commentId: {
+            [Op.eq]: comment.id
+          },
+          userId: {
+            [Op.eq]: userId
+          }
+        }
+      }).then((commentLike) => {
+        if (!commentLike) {
+          return res.status(400).json({
+            commentId: comment.id,
+            message: 'Comment has not been liked'
+          });
+        }
+
+        commentLike.destroy();
```

Comment:
Use `.then()` for `commentLike.destroy();`. So the response will be sent on actually delete.